details.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
+from catalog import models as cmod
+import math
+
+
+@view_function
+def process_request(request, prod:cmod.Product=None,):
+
+    # if prod.TITLE == 'Bulk':
+    #     qty = prod.quantity
+    #
+    # else:
+    #     qty = 0;
+
+
```

Comment:
you might want to add something like    
 if product is None:
        return HttpResponseRedirect('catalog/index/')
to handle no product being passed in

Reply:
Ahhhhh didn't even think of that. Thanks.